new-site/src/components/Table.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Table = (props) => (
+  <div className="hds-table-container docTable">
+    <table className="hds-table hds-table--dark" aria-label="Service users (dark variant)">
```

Comment:
Should this aria-label be configured through props? It probably is different depending on where the table is used?

Reply:
Mika can answer in detail but even if it could, how would one input the label since this table will be generated from Markdown?

Reply:
I guess it is just copy paste thing. I think we need to figure a way to pass the caption to markdown table.